frontend/src/components/SearchBar/FilterBar.scss

```diff
@@ -20,4 +20,8 @@
     width: 44px;
     height: 36px;
   }
+  display: flex;
+  flex-wrap: wrap;
+  margin-right: -5px;
+  margin-left: -5px;
 }
```

Comment:
these styles align the drop downs on the filter bar so they are aligned horizontally